classes/autoptimizeToolbar.php

```diff
@@ -0,0 +1,115 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+class autoptimizeToolbar {
+
+	public function __construct()
+	{
+		// If Cache is not available we don't add the Autoptimize Toolbar
+		if( !autoptimizeCache::cacheavail() ) return;
+
+		// Load admin toolbar feature once WordPress, all plugins, and the theme are fully loaded and instantiated.
+		add_action( 'wp_loaded', array( $this, 'load_toolbar' ) );
+	}
+
+	public function load_toolbar()
+	{
+		// We check that the current user has the appropriate permissions
+		if( current_user_can( 'manage_options' ) )
+		{
+			// Load custom styles and scripts
+			if( is_admin() ) {
+				// in the case of back-end
+				add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+			} else {
+				// in the case of front-end
+				add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+			}
+			
+			// Create a handler for the AJAX toolbar requests
+			add_action( 'wp_ajax_autoptimize_delete_cache', array( $this, 'delete_cache' ) );
+
+			// Add the Autoptimize Toolbar to the Admin bar
+			add_action( 'admin_bar_menu', array($this, 'add_toolbar'), 100 );
+		}
+	}
+
+	public function add_toolbar()
+	{
+		global $wp_admin_bar;
+
+		// Retrieve the Autoptimize Cache Stats information
+		$stats = autoptimizeCache::stats();
+
+		// Set the Max Size recommended for cache files
+		$max_size = 512 * 1024 * 1024;
```

Comment:
can you change this into:`
$max_size = apply_filters('autoptimize_filter_cachecheck_maxsize', 512 * 1024 * 1024);`

it's the same filter used as in the cachechecker, allows people to override this without having to edit our code :)